### DISCUSSION TITLE
[Script request] Borgwarehouse |A fast and modern WebUI for a BorgBackup

### DISCUSSION THREAD

**Discussion by mieslies** [community-scripts/ProxmoxVE#453] [Request script] [answered]
### Application Name

borgwarehouse 

### Website

https://borgwarehouse.com/

### Description

BorgWarehouse is a graphical interface to manage a central [BorgBackup](https://borgbackup.readthedocs.io/en/stable/#what-is-borgbackup) repository server. The BorgWarehouse, you have an interface that allows you to do all this simply and quickly :

- [ ]     add repositories
- [ ]     edit existing repositories
- [ ]     delete repositories
- [ ]     be alerted if there are no recent backups
- [ ]     monitor the volume of data
- [ ]     flexibly manage quotas for each repository

The whole system part is automatically managed by BorgWarehouse and you don’t have to touch your terminal anymore while enjoying a visual feedback on the status of your repositories. Borgwarehouse is primarily designed for self-hosting.

https://borgwarehouse.com/docs/admin-manual/debian-installation/

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment, marked as answer]
What do you need for this? What is it good for? What is the added value for the community?

**Reply by mieslies** [reply]
I have installed ProxMox backup server recently and that script is already available, this will do the job for backup my data. So please cancel my request. Thanks for the reply and your work on the scripts!
